pkgs/servers/x11/xorg/overrides.nix

```diff
@@ -1020,26 +1020,10 @@ self: super:
     meta = attrs.meta // { mainProgram = "xinit"; };
   });
 
-  xf86videointel = super.xf86videointel.overrideAttrs (attrs: {
-    # the update script only works with released tarballs :-/
-    name = "xf86-video-intel-2021-01-15";
-    src = fetchFromGitLab {
-      domain = "gitlab.freedesktop.org";
-      group = "xorg";
-      owner = "driver";
-      repo = "xf86-video-intel";
-      rev = "31486f40f8e8f8923ca0799aea84b58799754564";
-      sha256 = "sha256-nqT9VZDb2kAC72ot9UCdwEkM1uuP9NriJePulzrdZlM=";
-    };
-    buildInputs = attrs.buildInputs ++ [ xorg.libXScrnSaver xorg.libXv xorg.pixman xorg.utilmacros ];
-    nativeBuildInputs = attrs.nativeBuildInputs ++ [autoreconfHook ];
-    configureFlags = [ "--with-default-dri=3" "--enable-tools" ];
-    patches = [ ./use_crocus_and_iris.patch ];
-
-    meta = attrs.meta // {
-      platforms = ["i686-linux" "x86_64-linux"];
-    };
-  });
+  xf86videointel = throw ''
+    xf86videointel has been removed as the package is unmaintained and the driver is no longer functional.
+    Please remove "intel" from `services.xserver.videoDrivers` and switch to the "modesetting" driver.
+  ''; # Added 2024-12-16;
```

Comment:
This results in this error:
```
 error: xf86videointel has been removed as the package is unmaintained and the driver is no longer functional.
 Please remove "intel" from `services.xserver.videoDrivers` and switch to the "modesetting" driver.
```
if you specified `services.xserver.videoDrivers = [ "intel" ];`

Reply:
Should we also delete all the other DDXes while we're at it? I highly doubt anyone wants to use or maintain them.

Reply:
I'd like to get this merged asap, so the release notes are updated. Properly removing all the other DDX will take more time.